tests/admin_widgets/false_timezone_warning_templates/admin/base.html

```diff
@@ -0,0 +1,88 @@
+{% load i18n admin_static %}<!DOCTYPE html>
```

Comment:
Try to minimize the template as much as possible to only include the bits necessary for the test case. Also include an explanatory comment like "Intentionally missing data-.... in <body> tag."